dev/encoding/vqgan-jax-encoding-yfcc100m.ipynb

```diff
@@ -13,14 +13,14 @@
    "id": "ba7b31e6",
```

Comment:
<p>This can be useful but ideally we have only one notebook.</p><p>We can keep it for now if you want as I think we will have datasets that don't use the <code>datasets</code>  library but we can also create different getters for these (or we can build a loading script for them as well).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/borisdayma/dalle-mini/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Not sure either. I like how everything is clean and self-contained when using the <code>Datasets</code>  library, but I also like how fast the other notebook is. We should be able to achieve similar performance with these changes.</p>